### PR TITLE
DOC: Update getting started example, closes #662

### LINKED ISSUE #662: Deconvolution example in doc not updated
The example in the doc has not been updated to use functionals as per #569.

### PR DESCRIPTION
**Pull request by adler-j**

### CODE REVIEW

File: doc/source/guide/introduction/getting_started.rst [hunk 1]
```diff
@@ -206,13 +206,19 @@ Perhaps a bit better, but far from excellent.
...
 Since this is a non-differentiable problem we need more advanced solvers to solve it.
-One of the stronger solvers in ODL is the Douglas-Rachford Primal-Dual method (`douglas_rachford_pd`) which uses :ref:`proximal_operators` to solve the optimization problem.
+One of the stronger solvers in ODL is the Douglas-Rachford Primal-Dual method (`douglas_rachford_pd`) which uses :ref:`proximal_operators` to solve the optimization problem, but as a new user you do not need to consider the specifics, instead you only need to assemble the problem to fit the solver. 
+
+Consulting the `douglas_rachford_pd` documentation we see that it solves problems of the form:
+
+.. math::
+    \min_x f(x) + \sum_{i=0}^n g_i(L_i x),
+
+where :math:`f`, :math:`g_i` are convex functions, :math:`L_i` are linear `Operator`'s. By identification, we see that the above problem can be written in this form if we let `f` be the indicator function on :math:`[0, 1]`, :math:`g_1` be the squared l2 distance :math:`\| . - g\|_2^2`, :math:`g_2` be the norm :math:`\| . \|_1`, :math:`L_1` be the convolution operator and :math:`L_2` be the gradient operator.
```
Comment (kohr-h) [inline doc/source/guide/introduction/getting_started.rst]: Put the second sentence on a new line.

File: doc/source/guide/introduction/getting_started.rst [hunk 2]
```diff
@@ -206,13 +206,19 @@ Perhaps a bit better, but far from excellent.
...
    \min_{0 \leq f \leq 1} \|Af - g\|_2^2 + a \|\nabla f\|_1
 
 Since this is a non-differentiable problem we need more advanced solvers to solve it.
-One of the stronger solvers in ODL is the Douglas-Rachford Primal-Dual method (`douglas_rachford_pd`) which uses :ref:`proximal_operators` to solve the optimization problem.
+One of the stronger solvers in ODL is the Douglas-Rachford Primal-Dual method (`douglas_rachford_pd`) which uses :ref:`proximal_operators` to solve the optimization problem, but as a new user you do not need to consider the specifics, instead you only need to assemble the problem to fit the solver. 
+
+Consulting the `douglas_rachford_pd` documentation we see that it solves problems of the form:
+
+.. math::
+    \min_x f(x) + \sum_{i=0}^n g_i(L_i x),
```
Comment (kohr-h) [inline doc/source/guide/introduction/getting_started.rst]: i=1

File: doc/source/guide/introduction/getting_started.rst (hunk 1 above)
Comment (kohr-h) [inline doc/source/guide/introduction/getting_started.rst]: \cdot

**Comment by adler-j** [comment]
Fixed comments, merge after CI.

File: doc/source/guide/introduction/getting_started.rst [hunk 3]
```diff
@@ -222,27 +232,27 @@ e.g. `forward_backward_pd`, `chambolle_pock_solver`, etc in the ODL `examples/so
...
+   f = odl.solvers.IndicatorBox(space, 0, 1)
+   
    # Find scaling constants so that the solver converges.
    # See the douglas_rachford_pd documentation for more information.
    opnorm_A = odl.power_method_opnorm(A, xstart=g)
    opnorm_grad = odl.power_method_opnorm(grad, xstart=g)
    sigma = [1 / opnorm_A**2, 1 / opnorm_grad**2]
    tau = 1.0
-
+
```
Comment (kohr-h) [inline doc/source/guide/introduction/getting_started.rst]: ?

Comment (adler-j) [inline doc/source/guide/introduction/getting_started.rst]: Copy pasted from an editor, that one stays :D

**Pull request merged**